starters/nextjs-starter-approuter-ts/app/examples/ssg-isr/page.tsx

```diff
@@ -1,34 +1,56 @@
 import { PCCConvenienceFunctions } from "@pantheon-systems/pcc-react-sdk/server";
 import { Metadata } from "next";
-import { ArticleGrid } from "../../../components/grid";
+import ArticleList from "../../../components/article-list";
 import Layout from "../../../components/layout";
-import PageHeader from "../../../components/page-header";
+import { PAGE_SIZE } from "../../../constants";
 
 export const metadata: Metadata = {
   title: "SSG and ISR Example",
   description: "Example of using SSG and ISR",
 };
 
+async function fetchNextPages(cursor: string) {
+  "use server";
+  const { data, cursor: newCursor } =
+    await PCCConvenienceFunctions.getPaginatedArticles({
+      pageSize: PAGE_SIZE,
+      cursor,
+    });
+  return {
+    data,
+    newCursor,
+  };
+}
+
 export default async function SSGISRExampleTemplate() {
-  const articles = await PCCConvenienceFunctions.getAllArticles();
+  const {
+    data: articles,
+    cursor,
+    totalCount,
+  } = await PCCConvenienceFunctions.getPaginatedArticles({
+    pageSize: PAGE_SIZE,
+  });
 
   return (
     <Layout>
-      <section className="max-w-screen-3xl mx-auto px-4 pt-16 sm:w-4/5 md:w-3/4 lg:w-4/5 2xl:w-3/4">
-        <PageHeader title="SSG and ISR Example" />
```

Comment:
This page header is shown in the other starters so it should be shown here too

Reply:
`PageHeader` has been moved inside `ArticleList` component.